packages/flutterfire_ui/lib/src/auth/oauth/oauth_providers.dart

```diff
@@ -19,9 +67,7 @@ abstract class OAuthProvider {
     return credential;
   }
 
-  Future<void> signOut() async {
-    await FirebaseAuth.instance.signOut();
-  }
+  Future<void> signOut() async {}
```

Comment:
not really. The problem is that there is no way to figure out what provider was used to sign in, so we need to call `signOut` on all providers (email, phone, etc.), so that providers could execute their own cleanup logic, but only oauth providers have something to clean up, others have noop. We could make this method abstract, but that would require all implmentors have an empty sign out, so I decided to move this further down